_layouts/formula.html

```diff
@@ -30,40 +30,64 @@
 <p><a target="_blank" href="{{ site.taps.linux.remote }}/blob/HEAD/Formula/{{ f.name }}.rb">Linux formula code</a> on GitHub</p>
 {%- else %}
 <p><a target="_blank" href="{{ site.taps.core.remote }}/blob/HEAD/Formula/{{ f.name }}.rb">Formula code</a> on GitHub</p>
+{%- endif -%}
+
+{%- if f.versions.bottle %}
+<p>Bottle (binary package) installation support provided for
+    {%- if formula_path == "formula-linux" %} Linux platforms
+    {%- else %} macOS releases
+    {%- endif -%}:</p>
+<table>
+    {%- if f.bottle_disabled %}
```

Comment:
Is this just if it's disabled or if it's unneeded? Got any example screenshots for the latter (as we don't have any of the prior in homebrew-core)

Reply:
`bottle_disabled` is true if the formula specifies `bottle :unneeded`, which is shown in the third screenshot above.

Reply:
Gotcha, thanks. I'm just not sure about the `not applicable` here 🤔. Maybe "no bottle required", "no bottle needed", hide the entire section or have a completely reworded section. Thoughts?

Reply:
I'd prefer hiding it completely

Reply:
My reasoning for wanting to show something is so people can tell that we expect this to work on ARM Big Sur.

Reply:
I get that, we don't though. For non-bottled formula we have no record if it works.

Reply:
![after 4](https://user-images.githubusercontent.com/3324775/105366528-72f4c800-5bcd-11eb-8af6-41d19ea90e3c.png)?

Reply:
> I get that, we don't though. For non-bottled formula we have no record if it works.

1) the chance of it not working are very low
2) we do run all these through CI on every platform.

Reply:
I mean, I know where to find if it works. We just don't have a record like we do for bottles. 

> the chance of it not working are very low

Java software is often unbottled, and Java is a pain to get the build right. I'm not so sure this is true.